Debug.h

```diff
@@ -0,0 +1,34 @@
+#ifndef HEADER_Debug
+#define HEADER_Debug
+/*
+htop - Debug.h
+(C) 2020 htop dev team
```

Comment:
```suggestion
(C) 2025 htop dev team
```

Reply:
I didn't change any code here, so ended up just leaving the same copyright notice as the original file.